notebooks/official/workbench/spark/spark_bigquery.ipynb

```diff
@@ -0,0 +1,1221 @@
+{
```

Comment:
<p>If you want to include the output, feel free to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>